iceoryx2/src/lib.rs

```diff
@@ -493,16 +493,10 @@
 //! ```
 //!
 //! # Feature Flags
-//!
+//!  * `std` - Build for platforms that have `std` support
 //!  * `dev_permissions` - The permissions of all resources will be set to read, write, execute
 //!    for everyone. This shall not be used in production and is meant to be enabled in a docker
 //!    environment with inconsistent user configuration.
-//!  * `logger_console` - Include the [console logger](`iceoryx2-loggers::console::Logger`) and use it as the default logger
```

Comment:
Logger features removed from `iceoryx2` crate and moved to the `iceoryx2-loggers` crate.